geomstats/geometry/poincare_half_space.py

```diff
@@ -36,23 +36,26 @@ def __init__(self, dim, scale=1):
         self.metric = PoincareHalfSpaceMetric(self.dim, self.scale)
 
     def belongs(self, point):
-        """Test if a point belongs to the hyperbolic space.
+        """Evaluate if a point belongs to the manifold of normal distributions.
 
-        Test if a point belongs to the hyperbolic space based on
-        the poincare ball representation.
+        The statistical manifold of normal distributions is the upper
+        half plane.
```

Comment:
I don't think we should restrict the use of this function to information geometry. The Poincare upper half plane can be of interest in other contexts. I would write ```Poincare half space``` instead of ```manifold of normal distributions```.

Reply:
Yes you are right, I made a mistake changing this docstring